lib/templates/reconciliationText.js

```diff
@@ -92,8 +92,10 @@ class ReconciliationText {
   static read(handle) {
     if (!templateUtils.checkValidName(handle, this.TEMPLATE_TYPE)) return false;
     fsUtils.createTemplateFolders(this.TEMPLATE_TYPE, handle, true);
+
     // Config.json
     let templateConfig = fsUtils.readConfig(this.TEMPLATE_TYPE, handle);
+    let originalConfig = { ...templateConfig };
     templateConfig = this.#checkHandleAndNameInConfig(templateConfig, handle);
```

Comment:
[SUGGESTION] Should orginalConfig be initialised _after_ `#checkHandleAndNameInConfig` potentially populates the fields `handle`, `name_en`, and `name_nl`?

Reply:
You are right that `#checkHandleAndNameInConfig` can modify the config, and that' what we want to check, so originalConfig should be defined before.

We define originalConfig as soon as templateConfig is loaded, kept as a copy of how it is currently stored. Later on we check it how it is at the end to check if it has changed or not, and only if changed we write again